ajax/libs/filesize/3.5.6/filesize.js

```diff
@@ -0,0 +1,167 @@
+"use strict";
```

Comment:
Although the content looks same, but my diff & md5sum reports difference
files I download from npm:
```
6c75124acbf55786da221c4b03353daf  ./3.5.6/filesize.es6.js
5edc9424204684ed6ad0777093abd5fb  ./3.5.6/filesize.js
```
files in this PR:
```
7cb7dc343b941547fc1e28c7d93ba000  ./3.5.6/filesize.es6.js
f7468628e89bd7a83c183ade8b8bbaea  ./3.5.6/filesize.js
```

Can you check these file again. Lots of thanks!